src/bundles/ipfs-provider.js

```diff
@@ -297,7 +328,17 @@ const selectors = {
   /**
    * @param {State} state
    */
-  selectIpfsInitFailed: state => state.ipfs.failed
+  selectIpfsInitFailed: state => state.ipfs.failed,
+  /**
+   * @param {State} state
+   */
+  selectIpfsConnectionError: state => state.ipfs.connectionError,
+  /**
+   * @param {State} state
+   */
+  selectIpfsPendingFirstConnection: state => {
+    return state.ipfs.pendingFirstConnection
```

Comment:
💭  no need for the `{ return`, you can just use the arrow function to return directly